src/adapters/vfs/system.js

```diff
@@ -162,6 +162,17 @@ module.exports = (core) => {
 
       return watch;
     },
+    /**
+     * Get filesystem capabilities
+     * @param {String} file The file path from client
+     * @param {Object} [options={}] Options
+     * @return {Object[]}
+     */
+    capabilities: vfs => (file, options = {}) =>
+      Promise.resolve({
```

Comment:
Could you add spaces here between the properties and the values?

Reply:
I think we should add a rule to the eslint configuration so this can be auto-fixed :)

Reply:
@andersevenrud the `afterColon` property of [`key-spacing`](https://eslint.org/docs/latest/rules/key-spacing) defaults to `true`, so I'm not sure why that isn't happening already

Reply:
I'm using the recommended base ruleset [here](https://github.com/os-js/eslint-config/blob/master/index.js), so, yeah. Not sure what's going on there.